artifacts/definitions/Linux/Events/ProcessStatuses.yaml

```diff
@@ -0,0 +1,62 @@
+name: Linux.Events.ProcessStatuses
+description: |
+  This artifact collects New process status logs from the Linux kernel.
```

Comment:
How about this?

This artifact collects new processes created by non-root users from the Linux kernel audit events.